config/locales/en/errors.yml

```diff
@@ -1,6 +1,7 @@
 ---
 en:
   errors:
+    title_prefix: Error
```

Comment:
Should this have a colon in it `Error: ` so the title will be **Error: XYZ**

Reply:
If you look at the line where it's used in the `page_template_helper`:
```ruby
content_for(:head_title) { "#{prefix}: #{text}" }
```
The colon is added there. I think the colon is styling/formatting and therefore doesn't belong in the translation file.